<ui version="4.0" >
 <class>SlideshowConfig</class>
 <widget class="QWidget" name="SlideshowConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>405</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="minimumSize" >
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
     <property name="text" >
      <string>Folders:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>m_addDir</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="KPushButton" name="m_addDir" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Add Folder...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="KPushButton" name="m_removeDir" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Remove Folder</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>105</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="3" >
    <widget class="KListWidget" name="m_dirlist" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3" >
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QTimeEdit" name="m_slideshowDelay" >
       <property name="dateTime" >
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2000</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="currentSection" >
        <enum>QDateTimeEdit::HourSection</enum>
       </property>
       <property name="displayFormat" >
        <string comment="(qtdt-format) Please do not change the quotes (') and translate only the content of the quotes." >hh 'Hours' mm 'Mins' ss 'Secs'</string>
       </property>
       <property name="time" >
        <time>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="minimumSize" >
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
     <property name="text" >
      <string>P&amp;ositioning:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>m_resizeMethod</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QComboBox" name="m_resizeMethod" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Color:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>m_color</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="KColorButton" name="m_color" >
       <property name="color" >
        <color>
         <red>70</red>
         <green>90</green>
         <blue>130</blue>
        </color>
       </property>
       <property name="defaultColor" >
        <color>
         <red>70</red>
         <green>90</green>
         <blue>130</blue>
        </color>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="1" colspan="3" >
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_newStuff" >
       <property name="toolTip" >
        <string>Download new wallpapers</string>
       </property>
       <property name="text" >
        <string>Get New Wallpapers...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1" colspan="2" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>105</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Change images every:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>m_slideshowDelay</cstring>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>label</zorder>
  <zorder>m_addDir</zorder>
  <zorder>m_removeDir</zorder>
  <zorder>m_dirlist</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
